### ISSUE TITLE
Runtime in init_signals.dm, line 37: Cannot execute null.remove addiction points().

### ISSUE THREAD

**Issue by cacogen** [tgstation/tgstation#57065]
```
proc name: on nometabolism trait gain (/mob/living/carbon/proc/on_nometabolism_trait_gain)
src: Rebeckah Noton (/mob/living/carbon/human)
src.loc: the bronze floor (134,212,4) (/turf/open/floor/bronze)
call stack:
Rebeckah Noton (/mob/living/carbon/human): on nometabolism trait gain(Rebeckah Noton (/mob/living/carbon/human), "no_metabolism")
CallAsync(Rebeckah Noton (/mob/living/carbon/human), /mob/living/carbon/proc/on_nom... (/mob/living/carbon/proc/on_nometabolism_trait_gain), /list (/list))
Rebeckah Noton (/mob/living/carbon/human): SendSignal("addtrait no_metabolism", /list (/list))
Spooky Scary Skeleton (/datum/species/skeleton): on species gain(Rebeckah Noton (/mob/living/carbon/human), Human (/datum/species/human), 0)
Spooky Scary Skeleton (/datum/species/skeleton): on species gain(Rebeckah Noton (/mob/living/carbon/human), Human (/datum/species/human), 0)
Rebeckah Noton (/mob/living/carbon/human): set species(/datum/species/skeleton (/datum/species/skeleton), 1, 0)
Rebeckah Noton (/mob/living/carbon/human): set species(/datum/species/skeleton (/datum/species/skeleton), 1, 0)
the skeletal remains (/obj/effect/mob_spawn/human/skeleton): equip(Rebeckah Noton (/mob/living/carbon/human))
the skeletal remains (/obj/effect/mob_spawn/human/skeleton): create(null, null)
world: ImmediateInvokeAsync(the skeletal remains (/obj/effect/mob_spawn/human/skeleton), /obj/effect/mob_spawn/proc/cre... (/obj/effect/mob_spawn/proc/create))
the skeletal remains (/obj/effect/mob_spawn/human/skeleton): Initialize(1)
the skeletal remains (/obj/effect/mob_spawn/human/skeleton): Initialize(1)
Atoms (/datum/controller/subsystem/atoms): InitAtom(the skeletal remains (/obj/effect/mob_spawn/human/skeleton), /list (/list))
Atoms (/datum/controller/subsystem/atoms): InitializeAtoms(null, null)
Atoms (/datum/controller/subsystem/atoms): Initialize(188199)
Master (/datum/controller/master): Initialize(10, 0, 1)
```

**Comment by cacogen** [comment]
#57051

**Comment by dragomagol** [comment]
Closed by #57051
